advanced/config.yaml

```diff
@@ -0,0 +1,87 @@
+components:
+  schemas:
+    Advanced.Config:
+      description: |-
+        An optional container for the configuration of advanced features.
+      type: object
+      properties:
+        latency:
+          $ref: '#/components/schemas/Advanced.Config.Latency'
+        event:
+          $ref: '#/components/schemas/Advanced.Config.Event'
+        convergence:
+          $ref: '#/components/schemas/Advanced.Config.Convergence'
+
+    Advanced.Config.Latency:
+      description: |-
+        The optional container for latency metric configuration.
+      type: object
+      properties:
+        enable:
+          description: |-
+            True to enable latency metrics using timestamps.
+            Enabling this option could possibly change flow packet payloads as 
+            implementations may add instrumentation values to the flow
+            packet payload.
+          type: boolean
+          default: false
+        mode:
+          description: |-
+            Select the type of latency measurement. The different types of 
+            latency measurements are:
+            
+
+            store_forward:
+            The time interval starting when the last bit of the frame leaves the
+            sending port and ending when the first bit of the frame is seen on
+            the receiving port (LIFO).  This is based on the RFC 1242 standard.
+
+
+            cut_through:
+            The time interval starting when the first bit of the frame leaves
+            the sending port and ending when the first bit of the frame is seen
+            on the receiving port (FIFO).  This is based on the RFC 1242 
+            standard.
+          type: string
+          enum: 
+          - store_forward
+          - cut_through
+          default: store_forward
+
+    Advanced.Config.Convergence:
+      description: |-
+        The optional container for convergence analytics configuration.
+      type: object
+      properties:
+        enable:
+          description: |-
+            True to enable convergence analytics. 
+            Enabling this option could possibly change flow packet payloads as 
+            implementations may add instrumentation values to the flow
+            packet payload.
+          type: boolean
+          default: false
+
+    Advanced.Config.Event:
+      description: |-
+        The optional container for event configuration.
+      type: object
+      properties:
```

Comment:
Shouldn't we provide separate containers for separate events per https://github.com/open-traffic-generator/models/pull/95#discussion_r613108350 ?

Reply:
Also, not all events are tied to flow metrics (instrumentation signature).